<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>КупиДом</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Current_user">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пользователь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buy_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Купить</string>
    </property>
   </widget>
   <widget class="QLabel" name="Current_money">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Деньги</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="with_finishing_radbutton">
       <property name="text">
        <string>С отделкой</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="without_finishing_radButton">
       <property name="text">
        <string>Без отделки</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="small_home_radButton">
       <property name="text">
        <string>Маленький дом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="big_home_radButton">
       <property name="text">
        <string>Большой дом</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="Home_view">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>541</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="clear_filter_check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить фильтр</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="budget_checker">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Учитывая бюджет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
